### ISSUE TITLE
icon missing in Galaxies

### ISSUE THREAD

**Issue by Turecky** [MISP/MISP#8038]
### Expected behavior

All icons visible in Galaxy list view

### Actual behavior

Icons missing for:
1. Malpedia
2. Preventive Measure
3. CONCORDIA Mobile Modelling Framework - Attack Pattern

### Steps to reproduce

Exist in latest version. Looks like incorrect icon naming.
Suggestions:
```
Malpedia
fa-shield => fa-shield-alt

Preventive Measure
fa-shiled => fa-shield-virus
```

For `CONCORDIA Mobile Modelling Framework - Attack Pattern` have no suggestion because icon name ar missing. Class names have only `black fas fa-`

### Version

v2.4.151 (3b31026d2702482a5b53dbbfa53594d9ccaa4a0e)

### Operating System

Ubuntu

### Operating System version

20.04

### PHP version

7.4.3

### Browser

Chrome

### Browser version

96.0.4664.110

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by adulau** [comment]
Thanks for the feedback.

I just fixed the Concordia one - https://github.com/MISP/misp-galaxy/commit/b2cc6277cf6cd878d16925719378c949d68dd171

I'll update the others.

**Comment by Turecky** [comment]
Fixed #3990
